ramls/edge-orders.raml

```diff
@@ -45,70 +58,99 @@ types:
         description: API Key
         type: string
   post:
-    description: Validate that the API Key provided can be used to place an order
+    description: Validate that the API Key provided can be used to place an order. The response body depends on the Vendor (GOBI or other)
     responses:
        200:
         description: Success
         body:
-          application/xml:
-           example: "<test>POST - OK</test>"
+         application/xml:
```

Comment:
Just like responses from the order placement endpoint, these should not be prescribed by edge-orders...